### PR TITLE
report lcov file to coveralls using gh action

### PR DESCRIPTION
**Pull request by DenysGonchar**
Use [Coveralls GH action](https://github.com/marketplace/actions/coveralls-github-action) for reporting coverage

### CODE REVIEW

**Comment by mongoose-im** [comment]
[8974.1](https://travis-ci.org/esl/MongooseIM/jobs/752745595) / Erlang 23.0.3 / small_tests / 676711f155278738cd51f2fea64fb0c8c4857b87
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2993/8974/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2993/8974/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2021-01-04_06.04.16/lib.mongooseim.logs/run.2021-01-04_06.04.16/suite.log.html)

File: rebar.config.script [hunk 1]
```diff
@@ -84,24 +84,5 @@ MaybeFIPSSupport = fun(Config) ->
...
-                    CONFIG1
-            end;
-        _ ->
-            CONFIG
-    end
-end,
-
-Config0 = MaybeConfigureCoveralls(CONFIG),
-Config1 = SetupIncludedApps(Config0, EnvApps),
-MaybeFIPSSupport(Config1).
```
Comment (chrzaszcz) [inline rebar.config.script:107]: Could you explain in the commit message why it is not needed anymore?

**Review by chrzaszcz — COMMENTED** [review]
One question from me.

File: rebar.config.script (hunk 1 above)
Comment (DenysGonchar) [reply rebar.config.script:107]: done

Comment (DenysGonchar) [reply rebar.config.script:107]: one more note, let's first merge https://github.com/esl/rebar3_codecov/pull/3/files and switch back to the commit version selection for rebar3 codecov plugin before we merge this PR

**Comment by mongoose-im** [comment]
[8977.1](https://travis-ci.org/esl/MongooseIM/jobs/752849689) / Erlang 23.0.3 / small_tests / 257e579f6ace6b480cac4261f90d0990aeae5350
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2993/8977/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2993/8977/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2021-01-04_19.12.48/lib.mongooseim.logs/run.2021-01-04_19.12.49/suite.log.html)

**Comment by mongoose-im** [comment]
[8978.1](https://travis-ci.org/esl/MongooseIM/jobs/752977401) / Erlang 23.0.3 / small_tests / a6da662f967d21a3eef976197f02b8c899e705c7
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2993/8978/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2993/8978/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2021-01-05_11.36.32/lib.mongooseim.logs/run.2021-01-05_11.36.33/suite.log.html)

**Review by chrzaszcz — APPROVED** [review]
Looks good!

**Pull request merged**